elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -150,6 +154,7 @@ protected RequestScope(JsonApiDocument jsonApiDocument, RequestScope outerReques
         this.queryParams = Optional.empty();
         this.sparseFields = Collections.emptyMap();
         this.predicates = Collections.emptyMap();
+        this.pagination = Pagination.getDefaultPagination();
```

Comment:
@aklish Yes. This shouldn't be part of any other query other than get. I can add into this PR, or move to another PR so this is a scoped PR.